charts/rstudio-connect/templates/pvc.yaml

```
@@ -5,7 +5,7 @@ metadata:
   name: {{default (print (include "rstudio-connect.fullname" .) "-shared-storage" ) .Values.sharedStorage.name }}
   namespace: {{ $.Release.Namespace }}
   annotations:
-    "helm.sh/resource-policy": keep
+{{ .Values.sharedStorage.annotations | toYaml | indent 4}}
```

```suggestion
    {{- .Values.sharedStorage.annotations | toYaml | nindent 4}}
```
I personally prefer this pattern because it makes indentation look proper in the template and in the output (same elsewhere)